### ISSUE TITLE
Using regioninfo.dat to update server doesn't work in new update.

### ISSUE THREAD

**Issue by Orangenal-name** [Impostor/Impostor#353]
# Bug Report

## Base Information
- Windows 10
- Impostor-Server_1.3.0-ci.150_win-x64
- 2021.3.5s

## I confirm:
- [x] that I have searched for an existing bug report for this issue.

## Symptoms 

In the latest version of among us, it no longer saves what server you were using inside regioninfo.dat, and goes so far as deleting it if you put one in the place it should be

## Reproduction

Download the latest version of among us and attempt to launch it with your server. It should work the first time, then close it and launch it up again and it should no longer work until you reinstall it again.

**Comment by fatalterreur** [comment]
Same issue, regioninfo.dat does not exist anymore. Now it's in regioninfo.json.
I tried to modify it but without success

**Comment by AeonLucid** [comment]
This is known, the patcher has not been updated yet. I am not sure why 1.3.0 was "released".
fyi @js6pak

**Comment by gamer4life1** [comment]
Shoot, I have this issue too and I hope it gets resolved soon.

**Comment by JohnTranQUT** [comment]
@gamer4life1 I know of a solution to this problem.
Replace the contents of the regionInfo.json file replaced with this:
### For PC/Steam
```json
{
  "NHKLLGFLCLM": 0,
  "PBKMLNEHKHL": [
    {
      "$type": "DnsRegionInfo, Assembly-CSharp",
      "Fqdn": "YOUR IP ADDRES OR IMPOSTOR SERVER DOMAIN",
      "Name": "Impostor",
      "TranslateName": 1003
    }
  ]
}
```
### For Andriods Devices
```json
{
  "CurrentRegionIdx": 0,
  "Regions": [
    {
      "$type": "DnsRegionInfo, Assembly-CSharp",
      "Fqdn": "YOUR IP ADDRES OR IMPOSTOR SERVER DOMAIN",
      "Name": "Impostor",
      "TranslateName": 1003
    }
  ]
}

```
To softly revert back, simply change regions.
To revert everything back to before, delete the regionInfo.json file. Among Us will remake it with the correct settings.
Please note that there is no option to change from the defaulted UDP ports.

-AznDumbum#7449

**Comment by js6pak** [comment]
This is already a known issue and was already discussed in discord.

`regionInfo.dat` actually still works, but among us tries to convert it to the new format but fails, resulting in deletion of `regionInfo.dat` (while still loading it) which is a bug confirmed by forte, and is already fixed in android build (2021.3.9a)

`regionInfo.json` kinda works, but has few problems in current game version: its obfuscated - bound to a platform, doesn't support announcements, `DnsRegionInfo` doesn't support setting port yet (`StaticRegionInfo` does!)
https://github.com/Impostor/Impostor/pull/343#issuecomment-797350038

Workaround for now would be just merging #343

**Comment by gamer4life1** [comment]
Thanks @js6pak and @JohnTranQUT!
